<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogChoixTypeImportPDF</class>
 <widget class="QDialog" name="DialogChoixTypeImportPDF">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="checkBoxVaccin">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Vaccin</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxProjet">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Projet</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxRessource">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Ressource</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxPatient">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Patient</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxMedecin">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Medecin</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditStart">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditEnd">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPath">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SelectPath">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Path</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonGenerer">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Generer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonRetour">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>De</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
